### PR TITLE
README: update links

### PR DESCRIPTION
**Pull request by CaitinChen**

### CODE REVIEW

**Comment by CaitinChen** [comment]
@lilin90 PTAL

File: README.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ The TiDB User Groups (TUGs) are groups for facilitating communication and
 discovery of information related to topics that have long term relevance to
 large groups of TiDB users.
 
-See [pingcap/docs](https://pingcap.com/docs/) or
-[pingcap/docs-cn](https://pingcap.com/docs-cn/) for user documents. Also you can get
-help in [AskTUG.com(Chinese)](https://asktug.com/) if you met any problem.
+See [pingcap/docs](https://docs.pingcap.com/tidb/stable) or
+[pingcap/docs-cn](https://docs.pingcap.com/zh/tidb/stable) for user documents. Also you can get
+help in [AskTUG.com (Chinese)](https://asktug.com/) if you met any problem.
```
Comment (lilin90) [inline README.md:26]:
```suggestion
See TiDB documentation in [English](https://docs.pingcap.com/tidb/stable) or [Chinese](https://docs.pingcap.com/zh/tidb/stable). You can also get help in [AskTUG.com (Chinese)](https://asktug.com/) if you encounter any problem.
```

File: README.md [hunk 2]
```diff
@@ -61,8 +61,8 @@ See [`contributors`](./contributors/README.md) for details.
 ## Learning Resources
 
 Learning resources are collected in the [`learning-resources`](./learning-resources/README.md). Here you can find all the
-resources which helps you learn and contribute TiDB. For example, you can learn
-TiDB architecture though the following blogs:
+resources which help you learn and contribute to TiDB. For example, you can learn
```
Comment (lilin90) [inline README.md:64]:
```suggestion
resources which can help you learn and contribute to TiDB. For example, you can learn
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Comment by lilin90** [comment]
@zz-jason @winkyao PTAL~ Thanks!

**Comment by hi-rustin** [comment]
@lilin90 Now this PR belongs to your sig. You can approve it.

**Comment by CaitinChen** [comment]
@lilin90 Could you help add a "can-merge" label and merge this PR?

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**